src/methods.ts

```diff
@@ -87,14 +87,42 @@ export interface MessageAttachment {
   footer?: string;
   footer_icon?: string; // footer must be present
   ts?: string;
-  actions?: { // note: this is a very minimal definition of link buttons, interactive buttons, and message menus
-    type: string;
-    text?: string;
-  }[];
+  actions?: AttachmentAction[];
   callback_id?: string;
   mrkdwn_in?: ('pretext' | 'text' | 'fields')[];
 }
 
+export interface AttachmentAction {
+  id?: string;
+  confirm?: Confirmation;
+  data_source?: string;
+  min_query_length: number;
+  name: string;
```

Comment:
This field is optional, so I think it should be defined as `name?` instead of `name` (it's required only for standard actions, but it's not used with link buttons - see https://api.slack.com/docs/message-attachments#link_buttons)